redirects.ts

```diff
@@ -189,16 +189,25 @@ export const redirectsOptions: Options = {
             from: '/general/run-a-node/node-operator-overview', // #200 Restructure docs
         },
         {
-            to: '/node/mainnet/',
-            from: '/general/oasis-network/network-parameters', // #200 Restructure docs
+            to: '/node/network/mainnet',
+            from: [
+                '/general/oasis-network/network', // #200 Restructure docs, #1438 Reorganize network parameters
+                '/node/mainnet/', // #1438 Reorganize network parameters
+            ]
         },
         {
-            to: '/node/mainnet/upgrade-log',
-            from: '/general/run-a-node/upgrade-log', // #200 Restructure docs
+            to: '/node/reference/upgrade-logs/mainnet', // #1438 Reorganize network parameters
```

Comment:
Please make sure the redirects are ordered by alphabet according to the `to` field.